hack/e2e-node-test.sh

```diff
@@ -129,6 +129,6 @@ else
   # Test using the host the script was run on
   # Provided for backwards compatibility
   "${ginkgo}" --focus=$focus --skip=$skip "${KUBE_ROOT}/test/e2e_node/" --report-dir=${report} \
-    -- --alsologtostderr --v 2 --node-name $(hostname) --build-services=true --start-services=true --stop-services=true
+    -- --alsologtostderr --v 2 --node-name $(hostname) --disable-kubenet=true --build-services=true --start-services=true --stop-services=true
```

Comment:
I'm concerned that a bug in networking will not show up with local testing. As much as possible, we want the local and remote testing environment to be the same. WDYT?